api-tester/src/defaults/java/com/revenuecat/apitester/java/RedeemRCBillingPurchaseListenerAPI.java

```diff
@@ -0,0 +1,27 @@
+package com.revenuecat.apitester.java;
+
+import com.revenuecat.purchases.interfaces.RedeemRCBillingPurchaseListener;
+
+import kotlin.Unit;
+import kotlin.jvm.functions.Function1;
+
+@SuppressWarnings({"unused"})
+final class RedeemRCBillingPurchaseListenerAPI {
+    static void checkListener(RedeemRCBillingPurchaseListener listener,
+                              Function1<RedeemRCBillingPurchaseListener.ResultListener, Unit> startRedemption) {
```

Comment:
This API is probably not great in Java 😅... I could create another interface to be passed in as parameter to the `handleRCBillingPurchaseRedemption` instead... But felt like 3 interfaces was a lot... Wdyt?

Reply:
Hmmm yeah, not a fan, not only becuase of the `Function1`, but because of `Unit`

Reply:
Improved in https://github.com/RevenueCat/purchases-android/pull/1880/commits/2a16fffc73c1a805ce34a957295cf828ba4024c3. It adds another interface... But seemed like the best approach to make it look better. Lmk what you think!